junit-platform-launcher/src/main/java9/org/junit/platform/launcher/listeners/FlightRecordingListener.java

```diff
@@ -112,27 +120,38 @@ public void executionFinished(TestIdentifier test, TestExecutionResult result) {
 	@Name("org.junit.Test")
 	@Label("Test")
 	static abstract class TestEvent extends Event {
-
 		@Label("Unique Id")
 		String uniqueId;
-
 		@Label("Display Name")
 		String displayName;
 	}
 
 	@Name("org.junit.DynamicTestRegistration")
+	@Label("Dynamic Test Registration")
 	static class DynamicTestEvent extends TestEvent {
 	}
 
 	@Name("org.junit.SkippedTest")
+	@Label("Skipped Test")
 	static class SkippedTestEvent extends TestEvent {
 		@Label("Reason")
 		String reason;
 	}
 
 	@Name("org.junit.TestExecution")
+	@Label("Test")
```

Comment:
@Category("JUnit"

Reply:
To make it explicit? As `@Category` is already inherited from the super class.

Reply:
Sorry. I missed that. I'm not used to GitHub, is there a way I can see the files in their latest state without comments (on the web)?

Reply:
Thought there was a filter for that, but couldn't find it.

Try "View file..." in this drop down menu:
![image](https://user-images.githubusercontent.com/2319838/80278886-1f1a8880-86fa-11ea-9544-f57cfb08fcf1.png)

Reply:
Thanks!